src/components/HeadCellMenu/styles.ts

```diff
@@ -52,12 +52,13 @@ export const StyledDivider = styled(Divider)(({ theme }: DefaultTheme) => ({
   },
 }));
 
-export const StyledGroupLabel = styled(ListItem)(({ theme }: DefaultTheme) => ({
+export const StyledGroupLabel = styled(MenuItem)(({ theme }: DefaultTheme) => ({
   fontSize: '12px',
   height: HEAD_CELL_MENU_GROUP_LABEL_HEIGHT,
   fontWeight: 'bold',
   padding: theme.spacing(0.25, 2),
   marginBottom: theme.spacing(0.5),
+  opacity: '1 !important',
```

Comment:
The labels are type of `<MenuItem />`, so we get a tab stop by default through Mui a11y logic on dropdown (however it handls it internally), I tried below solutions for only labels to avoid having tab stop on them: 
1. tabindex={-1}
2. setting role="none"
3. having aria-hidden="true"
4. having those all together

did not fix the issue and mui constantly **still** focusing on the label when dropdown menu was opening!

So I accepted the defeat in this step and marked labels as disabled -> which in this case it avoid to focus on label but disabled prop on MenuItem reduces the opacity and makes it light gray.  So having `!important` makes everything happy!

Reply:
what if we add a class name, maybe `menu-section-label`, so that we get higher specificity than muis internal css rules? Not sure it works but worth trying

Reply:
inline is better